code/modules/clothing/head/helmet.dm

```diff
@@ -22,8 +22,8 @@
 	siemens_coefficient = 0.7
 	w_class = ITEM_SIZE_NORMAL
 	matter = /decl/material/solid/metal/steel
```

Comment:
two matter lists?

Reply:
Different prices for the items

Reply:
That's meant to be `material = /decl/material/solid/metal/steel`.

Reply:
Fixed didn't see I made two matters